alisa/org.osate.verify/src/org/osate/verify/linking/VerifyLinkingService.java

```diff
@@ -1,50 +0,0 @@
-package org.osate.verify.linking;
-
-import java.util.Collections;
-import java.util.List;
-
-import org.eclipse.emf.ecore.EClass;
-import org.eclipse.emf.ecore.EObject;
-import org.eclipse.emf.ecore.EReference;
-import org.eclipse.xtext.linking.impl.DefaultLinkingService;
-import org.eclipse.xtext.linking.impl.IllegalNodeException;
-import org.eclipse.xtext.nodemodel.INode;
-import org.eclipse.xtext.resource.IEObjectDescription;
-import org.osate.verify.verify.ResoluteMethod;
-
-import com.rockwellcollins.atc.resolute.resolute.ResolutePackage;
-
-public class VerifyLinkingService extends DefaultLinkingService {// PropertiesLinkingService {
```

Comment:
Please mark as deprecated and add a deprecation comment with a target version for removal. Otherwise we may forget to remove it.